tests/unit/hive_metastore_client/test_hive_mestastore_client.py

```diff
@@ -93,6 +94,36 @@ def test_add_columns_to_table(
             dbname=db_name, tbl_name=table_name, new_tbl=mocked_return_get_table
         )
 
+    @mock.patch.object(HiveMetastoreClient, "get_table")
+    @mock.patch.object(HiveMetastoreClient, "alter_table")
+    def test_drop_columns_from_table(
+        self, mocked_alter_table, mocked_get_table, hive_metastore_client
+    ):
+        # arrange
+        db_name = "db_name"
+        table_name = "table_name"
+        cols = ["col1", "col2"]
+
+        mocked_return_get_table = Mock()
+        mocked_return_get_table.sd.cols = [
+            FieldSchema(name="col1"),
+            FieldSchema(name="col2"),
+            FieldSchema(name="col3"),
+        ]
+        mocked_get_table.return_value = mocked_return_get_table
+
+        # act
+        hive_metastore_client.drop_columns_from_table(
+            db_name=db_name, table_name=table_name, columns=cols
+        )
+
+        # assert
+        mocked_get_table.assert_called_once_with(dbname=db_name, tbl_name=table_name)
+        mocked_alter_table.assert_called_once_with(
```

Comment:
I dont get how this test is working. The `new_tbl` arg from `mocked_alter_table` should not be equal `mocked_return_get_table` because the `new_table` should have only the column `col3`.

Reply:
As I could see running the test, pytest doesn't validate the Mock object properties.
This is the reason it's working (even with the `mocked_return_get_table` in `new_tbl` having the 3 columns) .
But, since you added the assert in line 125, I think it's ok =]

Reply:
I suggest we fix this, since it is wrong anyway :( 
I will add a commit to adjust this test